vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridViewIT.java

```diff
@@ -285,17 +285,25 @@ public void gridAsMultiSelect() {
         Assert.assertFalse(isRowSelected(grid, 6));
 
         // test the select all button
-        grid.findElement(By.id("selectAllCheckbox")).click();
+        TestBenchElement selectAllCheckbox = grid
+                .findElement(By.id("selectAllCheckbox"));
+        selectAllCheckbox.click();
         // deselect 1
         getCellContent(grid.getCell(0, 0)).click();
-        Assert.assertEquals("Select all should have been deselected", "false",
-                grid.findElement(By.id("selectAllCheckbox"))
-                        .getAttribute("aria-checked"));
```

Comment:
I guess this test was removed / refactoring in newer versions, updated the assertions here to include indeterminate state.